src/Css.elm

```diff
@@ -3171,6 +3209,200 @@ backgroundOrigins firstValue values =
+        { url : Supported
+        , linearGradient : Supported
+        , none : Supported
+        }
+    -> Style
+backgroundImage (Value value) =
+    AppendProperty ("background-image:" ++ value)
+
+
+{-| Sets [`background-image`](https://css-tricks.com/almanac/properties/b/background-image/) for multiple images.
+
+    backgroundImages
+        (linearGradient (stop red) (stop blue))
+        [ url "http://www.example.com/chicken.jpg" ]
+
+See also [`backgroundImage`](#backgroundImage) if you need only one.
+
+-}
+backgroundImages :
+    Value
+        { url : Supported
+        , linearGradient : Supported
+        }
+    ->
+        List
+            (Value
+                { url : Supported
+                , linearGradient : Supported
+                }
+            )
+    -> Style
+backgroundImages (Value first) rest =
+    let
+        peeled =
+            List.map (\(Value value) -> value) rest
+
+        values =
+            String.join "," (first :: peeled)
+    in
+    AppendProperty ("background-image:" ++ values)
+
+
+
+{- GRADIENTS -}
+
+
+{-| Sets [`linear-gradient`](https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient)
+
+    linearGradient (stop red) (stop blue) []
+
+    linearGradient (stop red) (stop blue) [ stop green ]
+
+See also [`linearGradient2`](#linearGradient2) if you don't need to set the angle.
+
+-}
+linearGradient :
+    Value { colorStop : Supported }
+    -> Value { colorStop : Supported }
+    -> List (Value { colorStop : Supported })
+    -> Value { provides | linearGradient : Supported }
+linearGradient (Value firstStop) (Value secondStop) moreStops =
+    let
+        peeledStops =
+            List.map (\(Value stop) -> stop) moreStops
+
+        stops =
+            String.join "," (firstStop :: secondStop :: peeledStops)
+    in
+    Value ("linear-gradient(" ++ stops ++ ")")
+
+
+{-| Sets [`linear-gradient`](https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient), providing an angle.
+
+    linearGradient2 (deg 90) (stop red) (stop blue) []
+
+    linearGradient2 (top left) (stop red) (stop blue) []
+
+See also [`linearGradient`](#linearGradient) if you don't need to set the angle.
+
+-}
+linearGradient2 :
+    Value
+        { to : Supported
+        , deg : Supported
+        , grad : Supported
+        , rad : Supported
+        , turn : Supported
+        }
+    -> Value { colorStop : Supported }
+    -> Value { colorStop : Supported }
+    -> List (Value { colorStop : Supported })
+    -> Value { provides | linearGradient : Supported }
+linearGradient2 (Value angle) (Value firstStop) (Value secondStop) moreStops =
+    let
+        peeledStops =
+            List.map (\(Value stop) -> stop) moreStops
+
+        stops =
+            String.join "," (firstStop :: secondStop :: peeledStops)
+    in
+    Value ("linear-gradient(" ++ angle ++ "," ++ stops ++ ")")
+
+
+{-| Provides a stop for a [gradient](https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient).
```

Comment:
Any reason not to link to [the CSS Tricks one](https://css-tricks.com/snippets/css/css-linear-gradient/) here?

Reply:
no reason whatsoever other than I didn't find it!

Reply:
although it can also be used for `radial-gradient`... but that's probably fine